orchestra/workflow/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class WorkflowError(Exception):
```

Comment:
i think we have core.errors for stuff like this.  that said, i'd entertain us distributing errors in the future.